doc-content/enterprise-only/openshift/startupstrategy-enable-proc.adoc

```diff
@@ -10,15 +10,23 @@ You
 endif::DM[]
 can scale the {KIE_SERVER} pod, but all the copies execute the same services.
 
-If you want to connect additional {KIE_SERVERS} to {CENTRAL}, you must enable the `OpenShiftStartupStrategy` setting in the environment. With this setting, {CENTRAL} automatically discovers {KIE_SERVERS} in the same namespace and these {KIE_SERVERS} can be configured to connect to the {CENTRAL}.
+You can connect additional {KIE_SERVERS} to {CENTRAL}. However, if you deployed a single authoring environment using the `{PRODUCT_INIT}{ENTERPRISE_VERSION_SHORT}-authoring.yaml`, you must enable the `OpenShiftStartupStrategy` setting in the environment. With this setting, {CENTRAL} automatically discovers {KIE_SERVERS} in the same namespace and these {KIE_SERVERS} can be configured to connect to the {CENTRAL}.
```

Comment:
Minor rewrite suggestion:

When `OpenShiftStartupStrategy` is enabled, {CENTRAL} automatically discovers {KIE_SERVERS} in the same namespace and these {KIE_SERVERS} can be configured to connect to the {CENTRAL}.